bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/MonitorThread.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+import java.util.Iterator;
+import java.util.Map.Entry;
+import org.openhab.core.events.EventPublisher;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class provide to read typical state and put it on Openhab bus events
+ * 
+ * @author Tonino Fazio
+ * @since 1.7.0
+ */
+public class MonitorThread extends Thread {
+
+	int REFRESH_TIME;
+	private SoulissTypicals soulissTypicalsRecipients;
+	private static Logger LOGGER = LoggerFactory.getLogger(MonitorThread.class);
```

Comment:
either add "final" oder respell to "logger" since only constants are written in capital letters